CookingRecipies/Components/homePage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styles from '../styles/styles.js';
+import Search from "./Search.js";
+import RecipeList from './RecipeList.js'
+
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  StatusBar,
+} from 'react-native';
+
+
+const HomePage = () => {
+  return (
+
+      <SafeAreaView
+      // style={{
+      //   flex: 1, 
+      //   justifyContent: "center", 
+      //   alignItems: "center"
+      //  }}
+       >
+        {/* <StatusBar barStyle="dark-content" /> */}
+        <Search/>
+        <Text style={styles.red}>Hello home page</Text>
+        {/* <View>
+
+        <RecipeList/>
+        </View> */}
+      </SafeAreaView>
+
+  );
+};
+
+HomePage.navigationOptions = {
+    title: 'Home',
```

Comment:
I don't know if this will cause any errors, but the trailing comma might need to be taken out.